go/mysql/connection.go

```diff
@@ -132,6 +132,7 @@ func (this *ConnectionConfig) GetDBUri(databaseName string) string {
 	connectionParams := []string{
 		"autocommit=true",
 		"interpolateParams=true",
+		"multiStatements=true",
```

Comment:
I think it would be better not to set this on all connections opened in `gh-ost`, but only for the `db` connection of the  `Applier` struct.

Reply:
This can be done in `InitDBConnections`.